policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/amount.yaml

```diff
@@ -4,6 +4,7 @@ values:
   2023-01-01: 1_000
 metadata:
   unit: currency-USD
+  period: year
   label: Oregon Child Tax Credit amount
   reference:
     - title: 2023R1 House Bill 3235 Section 2 (3)(a)
```

Comment:
is this on the 2023 tax form?

Reply:
same question for all ctc files

Reply:
> same question for all ctc files

In 2023 tax forms don't talk about this house bill, but the link was already here before I took over. Should I remove it?